src/main/java/Duke.java

```diff
@@ -1,10 +1,80 @@
+                System.out.println("Here are the tasks in your list: ");
+                for (int i = 0; i < tasks.size(); i++) {
+                    System.out.print(i + 1 + ".");
+                    System.out.println(tasks.get(i));
+                }
+            } else if (userInput.trim().startsWith("done")) {
+                int processedInput;
+                processedInput = Integer.parseInt(userInput.replaceAll("[^0-9]", "")) - 1;
+                tasks.get(processedInput).setDone();
+                System.out.println("Nice! I've marked this task as done: ");
+                System.out.println(tasks.get(processedInput));
+            } else if (userInput.startsWith("todo")) {
+                System.out.println("Got it. I've added this task: ");
+                Task newTask = new Todo(userInput.substring(5));
+                tasks.add(newTask);
+                System.out.println("  " + newTask.toString());
+                System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+            } else if (userInput.startsWith("deadline")) {
+                System.out.println("Got it. I've added this task: ");
+                String by = "";
+                String processedDeadlineInput;
+                int getSlashIndex = 0;
+                for (int i = 0; i < userInput.length(); i++) {
+                    char getSlash = userInput.charAt(i);
+                    if (getSlash == '/') {
+                        getSlashIndex = i;
+                        break;
+                    }
+                }
+                by = userInput.substring(getSlashIndex + 4);
+                processedDeadlineInput = userInput.substring(9, getSlashIndex).trim();
+                Task newTask = new Deadline(processedDeadlineInput, by);
+                tasks.add(newTask);
+                System.out.println("  " + newTask.toString());
+                System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+            } else if (userInput.startsWith("event")) {
+                System.out.println("Got it. I've added this task: ");
+                String at;
+                String processedEventInput;
+                int getSlashIndex = 0;
+                for (int i = 0; i < userInput.length(); i++) {
+                    char getSlash = userInput.charAt(i);
+                    if (getSlash == '/') {
+                        getSlashIndex = i;
+                        break;
+                    }
+                }
+                at = userInput.substring(getSlashIndex + 4).trim();
+                processedEventInput = userInput.substring(6, getSlashIndex).trim();
+                Task newTask = new Event(processedEventInput, at);
+                tasks.add(newTask);
+                System.out.println("  " + newTask.toString());
+                System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+            }
+
+            Output.printBorder();
```

Comment:
All variables, methods, boolean values follow the coding standard by use of appropriate casings! Looks neat :)